lib/hotel_booker.rb

```diff
@@ -0,0 +1,117 @@
+require 'pry'
+require_relative 'room'
+
+
+module Hotel
+  
+  class HotelBooker
+    
+    attr_accessor :rooms, :reservations, :blocks
+    
+    def initialize(num_of_rooms)
+      rooms = Hotel::Room.load_all(num_of_rooms)
+      @rooms = rooms
+      @reservations = []
+      @blocks = []
+      
+    end
+    
+    def print_rooms
```

Comment:
I'd avoid using `print` in the names of methods that return strings, since people might think it outputs the string to the terminal instead.